src/components/StatusChip.tsx

```diff
@@ -0,0 +1,79 @@
+import BlockIcon from '@mui/icons-material/Block';
+import CheckIcon from '@mui/icons-material/Check';
+import HourglassEmptyIcon from '@mui/icons-material/HourglassEmpty';
+import { Box, Typography } from '@mui/material';
+import { StandardCSSProperties } from '@mui/system';
+import { useTranslation } from 'react-i18next';
+import { NviCandidateStatus } from '../types/nvi.types';
+import { Ticket } from '../types/publication_types/ticket.types';
+
+const ticketColor = {
+  PublishingRequest: 'publishingRequest.main',
+  DoiRequest: 'doiRequest.main',
+  GeneralSupportCase: 'generalSupportCase.main',
+};
+
+interface TicketStatusChipProps {
+  ticket: Ticket;
+}
+
+export const TicketStatusChip = ({ ticket }: TicketStatusChipProps) => {
+  const { t } = useTranslation();
+
+  if (ticket.status === 'NotApplicable' || ticket.type === 'UnpublishRequest') {
+    return null;
+  }
+
+  const text = t(`my_page.messages.ticket_types.${ticket.status}`);
+
+  if (ticket.status === 'Completed') {
+    return <StatusChip text={text} icon="check" color={ticketColor[ticket.type]} />;
+  }
+
+  return <StatusChip text={text} icon={ticket.status === 'Closed' ? 'block' : 'hourglass'} color="secondary.dark" />;
+};
+
+interface NviStatusChip {
+  status: NviCandidateStatus;
+}
+
+export const NviStatusChip = ({ status }: NviStatusChip) => {
+  const { t } = useTranslation();
+
+  const text = t(`tasks.nvi.status.${status}`);
+
+  if (status === 'Approved') {
+    return <StatusChip text={text} icon="check" color="nvi.main" />;
+  }
+
+  return <StatusChip text={text} icon={status === 'Rejected' ? 'block' : 'hourglass'} color="secondary.dark" />;
```

Comment:
`StandardCSSProps` er nytt for meg🤔 Hvorfor ikke bare sende inn `string`? Her settes også fargen til det samme som fallback?

Reply:
Siden det er den typen som forventes av `bgcolor` is `sx`. Ser nå at det skulle vært `StandardCSSProperties['backgroundColor']` i stedet for `StandardCSSProperties['color']`, så sikker noe copy-paste som glapp her.. Skal fikse det

![bilde](https://github.com/user-attachments/assets/6da2dde4-c59a-4f10-8662-32a268603821)

Reply:
Når det gjelder at fallbacken blir det samme er jeg litt usikker på hva jeg liker best; eksplisitt gjengivelse eller bare lene seg på fallback. Men litt styr å passe på at det er `secondary.dark` og ikke `secondary.main` alle plasser feks, så tror jeg er mest enig i at det kan fjernes 😅 Tar vekk det også